release-to-candidate/action.yaml

```diff
@@ -43,23 +47,57 @@ runs:
         git config --global user.email "github-actions@github.com"
         git config --global user.name "Github Actions"
 
-        # Install moreutils so we have access to sponge
-        sudo apt-get update; sudo apt-get install -y moreutils
+    - name: Find the snapcraft.yaml path
+      id: yaml-path
+      shell: bash
+      run : |
+        if [[ -n "${{ inputs.snapcraft-yaml-path }}"; then
+          yaml-path="${{ inputs.snapcraft-yaml-path }}"
```

Comment:
Sorry, I appreciate I might have been confusing in a past comment, but you can't name `bash` variables with `-`.

Your Github inputs/outputs can use hyphens, but `bash` variable names need to use `_` or it's a syntax error

Reply:
Check if it's fine now..